README.md

```diff
@@ -5,24 +5,48 @@ NetCDF example files used for the testing suite of `xclim`
 In order to add a new dataset to the `xclim` testing data, please ensure you perform the following:
 
 1. Create a new branch: `git checkout -b my_new_testdata_branch`
-2. Place your dataset within an appropriate subdirectory (or create a new one: `mkdir data/my_testdata_contribution`).
-3. Run the registry generation script: `python make_check_sums.py`
+2. Place your dataset within an appropriate subdirectory (or create a new one: `mkdir data`).
+3. Run the sha256 checksum generation script: `python report_check_sums.py`
 4. Commit your changes: `git add testdata_contribution && git commit -m "added my_new_testdata"`
 5. Open a Pull Request.
 
 To modify an existing dataset, be sure to remove the existing checksum file before running the `make_check_sums.py` script.
 
 If you wish to perform preliminary tests against the dataset using `xclim`, this can be done with the following procedure:
+
+* To gather a single file:
```

Comment:
This example seems very similar to the next one ? Is there a reason to show both ? The first one is more generic, but I feel the second one is clear enough about what parts can be generalized!